panther_gazebo/src/led_strip_manager.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "panther_gazebo/led_strip_manager.hpp"
+
+LedStripManager::LedStripManager(const std::string & config_file)
+{
+  LoadConfig(config_file);
+  CreateLedStrips();
+}
+
+void LedStripManager::LoadConfig(const std::string & config_file)
+{
+  try {
+    config_ = YAML::LoadFile(config_file);
+  } catch (const std::exception & e) {
+    std::cerr << "Error loading configuration: " << e.what() << std::endl;
+    throw;
```

Comment:
```suggestion
    throw std::runtime_error("Error loading configuration: " + e.what());
```
Why not like this if we throw anyway?